### PR TITLE
Log Sentry errors

### LINKED ISSUE #51: Log Sentry errors with Monolog
https://docs.sentry.io/clients/php/usage/#handling-failures - it would be good to know when and why event was not reported to Sentry

### PR DESCRIPTION
**Pull request by duxet**
Resolves #51

### CODE REVIEW

**Comment by Addvilz** [comment]
Looks good, but please, add tests in appropriate places.

**Comment by duxet** [comment]
@Addvilz done :)

File: src/Sentry/SentryBundle/EventListener/ExceptionListener.php [hunk 1]
```diff
@@ -63,6 +66,14 @@ public function setClient(\Raven_Client $client)
     }
 
     /**
+    * @param \Psr\Log\LoggerInterface $logger
+    */
+    public function setLogger($logger)
```
Comment (Jean85) [inline src/Sentry/SentryBundle/EventListener/ExceptionListener.php]: Please typehint against the interface: `(LoggerInterface $logger = null)`

Comment (duxet) [inline src/Sentry/SentryBundle/EventListener/ExceptionListener.php]: I was sure that referencing class that might be not available will throw an exception, but i have checked that now and seems like this is not going to happen :)

**Comment by Addvilz** [comment]
@Jean85 perhaps leaving this disabled by default and adding a configuration option to provide logger service could be better? You can configure and pass whatever service that implements logger interface.

**Comment by Jean85** [comment]
Yes @Addvilz that is a good solution too.

**Comment by duxet** [comment]
I am working on some configuration options, i just need to figure out how it can be done.

**Comment by Jean85** [comment]
Maye you can use https://github.com/facile-it/sentry-psr-log ?

**Pull request closed without merging**